clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/model/GoogleAutoscalingPolicy.groovy

```diff
@@ -39,6 +39,12 @@ class GoogleAutoscalingPolicy {
   @ToString(includeNames = true)
   static class CpuUtilization {
     Double utilizationTarget
+    PredictiveMethod predictiveMethod
+
+    enum PredictiveMethod {
+      NONE,
+      STANDARD
```

Comment:
Do we expect there to be multiple predictive methods in the future, or should the UI simply be a checkbox for now?

Reply:
I think it's relatively likely there will be more in the future, but we can always change the the UI if they ever add that, so I'd say a checkbox for now...